models/vista3d/configs/inference.json

```diff
@@ -146,7 +147,7 @@
             {
                 "_target_": "Invertd",
                 "keys": "pred",
-                "transform": "@preprocessing",
+                "transform": "$copy.deepcopy(@preprocessing)",
```

Comment:
Any specific reason to deepcopy the preprocessing here?

Reply:
Hi @KumoLiu , do we have a related ticket in MONAI core?

Reply:
I created one: https://github.com/Project-MONAI/MONAI/issues/8056